rotkehlchen/exchanges/binance.py

```diff
@@ -893,6 +916,92 @@ def query_online_trade_history(
+    ) -> Optional[NamedTuple]:
+        """Processes a single deposit/withdrawal from binance and deserializes it
+
+        Can log error/warning and return None if something went wrong at deserialization
+        """
+        try:
+            if 'status' in raw_data and raw_data['status'] == 'Successful':
+                asset = asset_from_binance(raw_data['fiatCurrency'])
+                tx_id = get_key_if_has_val(raw_data, 'orderNo')
+                timestamp = deserialize_timestamp_from_binance(raw_data['createTime'])
+                fee = Fee(deserialize_asset_amount(raw_data['totalFee']))
+                link_str = str(tx_id) if tx_id else ''
+                amount = deserialize_asset_amount_force_positive(raw_data['amount'])
+                if transaction_type in ['deposit', 'withdrawal']:
+                    category = (AssetMovementCategory.DEPOSIT
+                                if transaction_type == 'deposit' else
+                                AssetMovementCategory.WITHDRAWAL)
+                    return AssetMovement(
+                        location=self.location,
+                        category=category,
+                        address=deserialize_asset_movement_address(raw_data, 'address', asset),
+                        transaction_id=tx_id,
+                        timestamp=timestamp,
+                        asset=asset,
+                        amount=amount,
+                        fee_asset=asset,
+                        fee=fee,
+                        link=link_str,
+                    )
+                else:
+                    crypto_asset = asset_from_binance(raw_data['cryptoCurrency'])
+                    obtain_amount = deserialize_asset_amount_force_positive(
+                        raw_data['obtainAmount'],
+                    )
+                    rate = deserialize_price(raw_data['amount'])
+                    if transaction_type == 'buy':
+                        order_type = TradeType.BUY
+                        base_asset = asset
+                        quote_asset = crypto_asset
+                    else:
+                        order_type = TradeType.SELL
+                        base_asset = crypto_asset
+                        quote_asset = asset
+                    return Trade(
+                        timestamp=timestamp,
+                        location=self.location,
+                        base_asset=base_asset,
+                        quote_asset=quote_asset,
+                        trade_type=order_type,
+                        amount=obtain_amount,
+                        rate=rate,
+                        fee=fee,
+                        fee_currency=base_asset,
+                        link=link_str,
+                    )
+
+        except UnknownAsset as e:
+            self.msg_aggregator.add_warning(
+                f'Found {str(self.location)} deposit/withdrawal with unknown asset '
```

Comment:
Here and in the other log messages prepend fiat to deposit/withdrawal 

```suggestion
                f'Found {str(self.location)} fiat deposit/withdrawal with unknown asset '
```